File: 240105/특정 문자로 시작하는 문자열/strings-that-start-with-a-specific-character.py
n = int(input())
word = [] 
for i in range(n):
    word.append(input())
word2 = input()
cnt = 0 
tal = 0 
for i in range(n):
    if word[i][0] == word2:
        cnt+=1
        tal = len(word[i]) + tal
print(f"{cnt} {tal/cnt:.2f}")

File: 240105/특정 문자로 시작하는 문자열/strings-that-start-with-a-specific-character.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        // 여기에 코드를 작성해주세요.
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        String[] word = new String[20];
        int tal = 0 ;
        int val = 0 ;
        for(int i = 0 ; i<n; i++){
            word[i] = sc.next();
        }
        char car = sc.next().charAt(0);

        for (int i = 0 ; i<n; i++){
            if (word[i].charAt(0) == car){
                tal++;
                val = word[i].length()+val;
            }
        }
        double aver = (double) val / tal;

        System.out.printf("%d %.2f",tal,aver);
    }
}